### DISCUSSION TITLE
After this change, when adding a dashboard schedule, dashboard names are seen as "Dashboard <1>" etc... with no name at all and hard to select a dashboard for a schedule from its ID.... Is not there a better option other than this?

### DISCUSSION THREAD

**Discussion by mfyuce** [apache/superset#18294] [Q&A / Help]
After this change, when adding a dashboard schedule, dashboard names are seen as "Dashboard <1>" etc... with no name at all and it is hard to select a dashboard for a schedule from its ID.... Is not there a better option other than this?

![image](https://user-images.githubusercontent.com/41781/117729566-b19da500-b1f3-11eb-8d49-b9c9bc19651a.png)

_Originally posted by @mfyuce in https://github.com/apache/superset/pull/11234#r629709595_

**Comment by dpgaspar** [comment]
Hi @mfyuce,

Please make sure to fill out the issue template, we need more detailed info in order to help you. Also note that this implementation of alerts and reports is deprecated please migrate to the new one.
